python3.html.markdown

```diff
@@ -4,6 +4,7 @@ contributors:
     - ["Louie Dinh", "http://pythonpracticeprojects.com"]
     - ["Steven Basart", "http://github.com/xksteven"]
     - ["Andre Polykanine", "https://github.com/Oire"]
+	- ["Zachary Ferguson", "http://github.com/zfergus2"]
```

Comment:
Could you remove the extra whitespace here, please? It might mess up the website generator.